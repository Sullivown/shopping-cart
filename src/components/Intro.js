import React from 'react';
import WeBuyImg from '../assets/images/we-buy.png';
import WeSellImg from '../assets/images/we-sell.png';
import WeTradeImg from '../assets/images/we-trade.png';

function Intro() {
	return (
		<div className='home-section light-bg intro'>
			<section className='intro-section'>
				<img src={WeBuyImg} alt='We buy' />
				<h3>We Buy</h3>
				<p>
					We buy your old unwanted adventuring items. Get in touch for
					a valuation!
				</p>
			</section>
			<section className='intro-section'>
				<img src={WeSellImg} alt='We sell' />
				<h3>We Sell</h3>
				<p>
					We sell a wide range of adventuring items in our store.
					Please visit us to view our full range.
				</p>
			</section>
			<section className='intro-section'>
				<img src={WeTradeImg} alt='We trade' />
				<h3>We Trade</h3>
				<p>
					We also trade in unwanted items and treasures for store
					credit. Typically you will receive 20-30% more than cash
					value.
				</p>
			</section>
		</div>
	);
}

export default Intro;
