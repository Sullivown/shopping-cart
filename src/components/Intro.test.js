import React from 'react';
import '@testing-library/jest-dom';
import TestRenderer from 'react-test-renderer';

import Intro from './Intro';

describe('Test Intro component', () => {
	it('renders', () => {
		const introTestRenderer = TestRenderer.create(<Intro />).toJSON();
		expect(introTestRenderer).toMatchSnapshot();
	});
});
