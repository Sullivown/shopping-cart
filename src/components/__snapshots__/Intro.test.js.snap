// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Intro component renders 1`] = `
<div
  className="home-section light-bg intro"
>
  <section
    className="intro-section"
  >
    <img
      alt="We buy"
      src="we-buy.png"
    />
    <h3>
      We Buy
    </h3>
    <p>
      We buy your old unwanted adventuring items. Get in touch for a valuation!
    </p>
  </section>
  <section
    className="intro-section"
  >
    <img
      alt="We sell"
      src="we-sell.png"
    />
    <h3>
      We Sell
    </h3>
    <p>
      We sell a wide range of adventuring items in our store. Please visit us to view our full range.
    </p>
  </section>
  <section
    className="intro-section"
  >
    <img
      alt="We trade"
      src="we-trade.png"
    />
    <h3>
      We Trade
    </h3>
    <p>
      We also trade in unwanted items and treasures for store credit. Typically you will receive 20-30% more than cash value.
    </p>
  </section>
</div>
`;
